rest/urls.py

```diff
@@ -76,4 +80,19 @@
         RepoMemberUserGroupDetail.as_view(),
         name='repo-members-user-group-detail'
     ),
+    url(r'^repositories/(?P<repo_slug>[-\w]+)/learning_resources/$',
```

Comment:
Suggestion:

There are four URLs which have `r'^repositories/(?P<repo_slug>[-\w]+)/learning_resources/'` in common. Maybe that could be a variable, for clarity, like [this](https://github.com/mitodl/lore/blob/master/rest/urls.py#L23).